src/main/java/Ui.java

```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
+
+public class Ui {
+
+    Scanner in;
+
+    public Ui() {
+        in = new Scanner(System.in);
+    }
+
+    static String logo = " ____        _        \n\t"
+            + "|  _ \\ _   _| | _____ \n\t"
```

Comment:
I think this is generally good! You can consider putting the logo and the horizontalLines as a final variable since they wont be changing!